### PR TITLE
feat: upgrade to quince

### PR DESCRIPTION
**Pull request by Talha-Rizwan**

### CODE REVIEW

**Comment by Faraz32123** [comment]
Hi @Talha-Rizwan, thanks for the PR. Can u please resolve the merge conflicts.

**Comment by Faraz32123** [comment]
@Talha-Rizwan, can u please also update `install_requires` in `setup.py` file.
I think there's still some rebasing issue with this branch.

**Comment by Faraz32123** [comment]
@regisb, I guess we need to rebase this branch on the top of main branch right?
due to this `This branch cannot be rebased due to conflicts`
@Talha-Rizwan

**Comment by regisb** [comment]
Actually, the quince branch needs to be rebased on top of the nightly branch. And this might be obvious but: let's not merge this until Quince is out :sweat_smile:

File: tutorcredentials/__about__.py [hunk 1]
```diff
@@ -1,2 +1,8 @@
-__version__ = "16.0.3"
+__version__ = "17.0.0"
+__package_version__ = __version__
```
Comment (regisb) [inline tutorcredentials/__about__.py:2]: Remove everything including and below this line.

**Comment by regisb** [comment]
Hey @Talha-Rizwan it appears that the credentials image fails to build on quince (because there is no open-release/quince.master branch in openedx-i18n, which should not matter because credentials doesn't need this repo, but the build fails nonetheless).

It's really important that you thoroughly test _that your PR works_. In the meantime, I have to disable credentials on the demo server to deploy it successfully.

**Pull request closed without merging**